specification/web/resource-manager/Microsoft.Web/2016-08-01/WebApps.json

```diff
@@ -13936,6 +16885,10 @@
         }
       }
     },
+    "Object": {
```

Comment:
For now, I've hand-edited the WebApps.json swagger to remove the "Object" class and all references to it.  It should be in the most recent push.